### ISSUE TITLE
A0 adc limit input?

### ISSUE THREAD

**Issue by johndoe71rus** [nodemcu/nodemcu-devkit-v1.0#28]
what are the limits of the voltage A0? Usual esp-12x up to 1v, you use a resistive divider at the input.
can i measure li-ion battery voltage?

**Comment by vowstar** [comment]
No, you can't use it to measure li-ion battery directly. You'd better use an amplifier to convert the voltage.
